ovirt-imageio/pkg/v1beta1/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
and the sources within `ovirt-imageio/pkg` should move to `pkg/X` (pkg/ovirt-populator?)